### PR TITLE
style(frontend): create new button group for Hero [GIX-2953]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We change the buttons in the Hero to be grouped. Simultaneously we change the style.

### Before

Screenshot 2024-09-20 at 12 44 38
Screenshot 2024-09-20 at 12 44 44

### After

Screenshot 2024-09-18 at 12 41 00
Screenshot 2024-09-18 at 12 41 07
Screenshot 2024-09-18 at 12 41 13
Screenshot 2024-09-18 at 12 41 58

### CODE REVIEW

**Comment by peterpeterparker** [comment]
I'm confuse, should I review it or not? It's a draft without description but, you pinged me?

**Comment by AntonioVentilii-DFINITY** [comment]
Very sorry, still a draft, it was a fat finger click...

File: src/frontend/src/lib/components/ui/HeroButtonGroup.svelte [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div
+	role="toolbar"
+	class="toolbar w-full px-4 py-6 text-black inline-flex justify-between rounded-[32px] bg-[var(--color-primary)]"
+>
+	<slot />
+</div>
+
+<style lang="scss">
+	.toolbar > :global(:last-child) {
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/ui/HeroButtonGroup.svelte:9]: Is there another way? Tailwing has a built in `last-child` selector I think for example

File: src/frontend/src/lib/components/hero/Actions.svelte [hunk 2]
```diff
@@ -31,44 +32,46 @@
...
-	{/if}
-
-	{#if convertEth}
+<div role="toolbar" class="flex w-full justify-center pt-10 pb-3">
+	<HeroButtonGroup>
 		{#if $networkICP}
-			<ConvertToEthereum />
-		{:else}
-			<ConvertToCkETH />
+			<IcReceive token={$tokenWithFallback} />
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/hero/Actions.svelte:38]: No logical changes right? The GitHub view add an empty block so just want to be sure

File: src/frontend/src/lib/components/ui/HeroButtonGroup.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/ui/HeroButtonGroup.svelte:9]: i tried that and it was not working, but I am don't remember why. Let me re-try

File: src/frontend/src/lib/components/hero/Actions.svelte (hunk 2 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/hero/Actions.svelte:38]: yes, correct! I am not really sure why Github shows it like that, but the only change is that I wrap everything in `HeroButtonGroup`. My IDE view:

Screenshot 2024-09-20 at 13 55 20
Screenshot 2024-09-20 at 13 55 30

File: src/frontend/src/lib/components/ui/HeroButtonGroup.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/ui/HeroButtonGroup.svelte:9]: So, as I remembered: `last:border-r-0` or  `last:border-0` or `last:border-none` do not work... not even `last:!border-0`

I am not really sure why, I cannot find any documentation about it, but I have to imagine that it has something to do with the cascading of CSS

any suggestion?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/ui/HeroButtonGroup.svelte:9]: ok... found the solution... i was being stupid... the class goes inside `ButtonHero` and not in the button group component

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**